### ISSUE TITLE
Change default tokenize

### ISSUE THREAD

**Issue by jadb** [UseMuffin/HyperlinkAuth#1]
Ref: https://twitter.com/NeoThermic/status/712986560375992320

**Comment by ADmad** [comment]
I think `Security::randomBytes()` is available only since Cake 3.2. So we need to either bump Cake version or add fallback.

**Comment by jadb** [comment]
Good point. Fallback I think is better; there are still many downloads of the previous versions every day.
